src/mixed-line-bar-chart/__integ__/detail-popover-series-content.test.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { setupTest } from './common';
+import createWrapper from '../../../lib/components/test-utils/selectors';
+
+describe('Detail popover series content', () => {
+  describe('keeps expanded state independently for every datum', () => {
+    test.each(['hover', 'click'])('on %s', interaction =>
+      setupTest('#/light/mixed-line-bar-chart/drilldown?expandableSubItems=true', async page => {
+        const textUnderFirstExpandableSection = 'AWS Config';
+        const textUnderSecondExpandableSection = 'Amazon Elastic Container Service';
+        const wrapper = createWrapper().findMixedLineBarChart();
+        let currentIndex: number | undefined;
+
+        const goToDatum = (index: number, interaction: string) => {
```

Comment:
Why does this code not follow our standard practices of creating a sub class for the base page object with page specific interaction methods?

Reply:
Refactored [here](https://github.com/cloudscape-design/components/pull/1644/commits/3c067a09fbaca7e4e237a5c75e419b75cd4b3138)